### DISCUSSION TITLE
Q: NavigationView getting started

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [lepoco/wpfui#957] [Q&A]
I see there are a lot of interests in the `NavigationView`, however, I am not too clear, what does `NavigationView` gain me? It includes a menu bar? Toward which end? There are navigation pages? Are there any gallery examples how this thingy works? Why for instance do I need the nav view menu over a _"conventional"_ menu? Thanks...

**Comment by Mithicor** [comment]
In the Microsoft Store, search for "wpf-ui" and you can run the gallery app they have. It has several examples of the built in controls.

And yes, there are navigation pages. Depending on the navigation menu item you have selected, a page is displayed in the main content area.
